website/templates/project/settings.mako

```diff
@@ -92,6 +92,27 @@
                       <span data-bind="if: disabled" class="help-block">
                         A top-level project's category cannot be changed
                       </span>
+
+                        <div class="form-group">
+                            <label for="title">Title:</label>
+                            <input class="form-control" type="text" maxlength="200" data-bind="value: title,
+                                                                                                      valueUpdate: 'afterkeydown'">
+                            <span class="text-danger" id="titleInputMessage"></span>
+                        </div>
+                        <div class="form-group">
+                            <label for="description">Description:</label>
+                            <textarea data-bind="value: description,
+                                             valueUpdate: 'afterkeydown'",
+                            class="form-control resize-vertical"></textarea>
+                            <span class="text-danger" id="descriptionInputMessage"></span>
+                        </div>
+                            <button data-bind="css: {disabled: !dirtyTitleDescription()},
+                                           click: updateTitle"
+                            class="btn btn-primary">Save Changes</button>
+                            <button data-bind="css: {disabled: !dirtyTitleDescription()},
+                                           click: function() { cancelUpdateTitle(); cancelUpdateDescription() }"
```

Comment:
Might make sense to have a `cancel` or `cancelAll` function that has the logic for canceling the whole form. This will make it easier to add more logic in the future , e.g. if we add more fields to the form.